check_product_qty_with_material/README.rst

```diff
@@ -0,0 +1,61 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+=====================
+Product Bom Qty Check
+=====================
+
+This module extends the functionality of ... to support ...
```

Comment:
Reports to display the quantity of finished good that can be produced based on the quantity on stock of the raw materials.